README.md

```diff
@@ -68,7 +68,10 @@ pip install -U lib/*/*.whl
 ```
 Install dependencies like *lxml* and *pyvmomi* from PyPI as other requirements were installed from SDK's lib directory
 ```cmd
-pip install -U <SDK_DIRECTORY>
+pip install -U <SDK_DIRECTORY_PATH>
+eg:pip install -U /root/vsphere-automation-sdk-python
+Or
+pip install -U <sdk.zip>
```

Comment:
```cmd
pip install -U <SDK_DIRECTORY_PATH>
```
Where <SDK_DIRECTORY_PATH> is either install directory of the SDK or location of SDK's zip
e.g. 
```cmd 
pip install -U /root/vsphere-automation-sdk-python
Or
pip install -U vsphere-automation-sdk-python-8.0.0.0.zip
```